user.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+# my files
+from connection import conect
+from client_functions import orders
+from Strategies import rolling_high
+
+
+class user():
+    def __init__(self,API_KEY, API_SECRET):
+        self.bfx = conect(API_KEY, API_SECRET)
+        #self.wallet = read_wallet()
```

Comment:
Why is this commented

Reply:
its an old version, the function wasnt available yet